modules/dataplex-datascan/README.md

```diff
@@ -380,15 +380,46 @@ module "dataplex-datascan" {
 
 ## IAM
 
-IAM is managed via several variables that implement different features and levels of control:
+IAM is managed via several variables that implement different levels of control:
```

Comment:
Can you update the description in all modules that use the IAM interface? We have intentionally aligned them and I would like to avoid having a few modules diverge...

Reply:
I would do all in one go or we will start drifting :)